frontend/freebase2wikidata.js

```diff
@@ -260,7 +260,56 @@ $(function() {
     var title = mw.config.get('wgTitle');
     return qidRegEx.test(title) ? title : false;
   }
+  
+ function generateNav() {
+    $('#p-ps-navigation').remove();
+    $('#mw-panel').append('<div class="portal" role="navigation" id="p-ps-navigation" aria-labelledby="p-ps-navigation-label"><h3 id="p-ps-navigation-label">PS navigation tool</h3></div>');
+    $('#p-ps-navigation').append('<div class="body"><ul id="p-ps-nav-list"></ul></div>');
+    
+    var b = [];
```

Comment:
Variables should have meaningful names + as we discussed I suggest replacing 2 arrays with an array of objects.